keyboards/planck/keymaps/csc027/rules.mk

```diff
@@ -0,0 +1,15 @@
+API_SYSEX_ENABLE = no
+AUDIO_ENABLE = yes
+BACKLIGHT_ENABLE = no
+BLUETOOTH_ENABLE = no
+BOOTMAGIC_ENABLE = no
+COMMAND_ENABLE = no
+CONSOLE_ENABLE = no
+EXTRAKEY_ENABLE = no
+LINK_TIME_OPTIMIZATION_ENABLE = yes
+MIDI_ENABLE = no
+MOUSEKEY_ENABLE = yes
+NKRO_ENABLE = yes
+RGBLIGHT_ENABLE = no
+SLEEP_LED_ENABLE = no
+UNICODE_ENABLE = no
```

Comment:
Assuming Planck rev6, this can be condensed down to:
```c
BOOTMAGIC_ENABLE = no
COMMAND_ENABLE = no
CONSOLE_ENABLE = no
EXTRAKEY_ENABLE = no
LTO_ENABLE = yes
RGBLIGHT_ENABLE = no
```

Reply:
Per both your suggestions, the Planck and Iris rules files have been condensed.